Weather/HourlyWeatherCell.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  HourlyWeatherCell.swift
+//  Weather
+//
+//  Created by admin on 04.05.2022.
+//
+import Foundation
+import UIKit
+
+class HourlyWeatherCell: UICollectionViewCell {
+    
+    static var reuseId: String = "HourlyWeatherCell"
+    
+    let weatherImage = UIImageView()
+    let temperature = UILabel()
+    let time = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = UIColor(white: 1, alpha: 1)
+        
+        setupElements()
+        setupConstraints()
+        
+        self.layer.cornerRadius = 4
+        self.clipsToBounds = true
+        
+    }
+    
+    func getDay(value: Int) -> String {
```

Comment:
I think that it necessary to modify the code by removing the calculation of the days of the week